lib/adf_variable_defaults.yaml

```
@@ -185,6 +192,87 @@ SO2:
 SOAG:
   category: "Aerosols"
 
+BC:
+  colormap: "RdBu_r"
+  diff_colormap: "BrBG"
+  scale_factor: 1000000000
+  add_offset: 0
+  new_unit: "ug/m3"
```

Certainly not required but is it possible to replace the `u` with the greek letter mu?  For example by doing something like this:

```suggestion
  new_unit: "$\mu$g/m3"
```

Not worth spending a lot of time on it, but if easy to do it might help make the ADF plots more "publication" ready by default.